db/migrate/20190607014201_add_column_approved_to_institutions.rb

```diff
@@ -0,0 +1,10 @@
+class AddColumnApprovedToInstitutions < ActiveRecord::Migration
+  def change
+    add_column :institutions, :approved, :boolean, null: false, default: false
+    reversible do |dir|
+      dir.up do
+        Institution.update_all(approved: true)
```

Comment:
This could end up in [broken migrations](https://github.com/rubocop-hq/rails-style-guide#define-model-class-migrations) if the model name changes. Something like:

```rb
# Define `table_name` in a custom named class to make sure that you run on the
# same table you had during the creation of the migration.
# In future if you override the `Product` class and change the `table_name`,
# it won't break the migration or cause serious data corruption.

class InstitutionModel < ActiveRecord::Base
  self.table_name = :institutions
end

# ...

  def change 
# ...
      dir.up do
        InstitutionModel.update_all(approved: true)
      end
# ...
```

cc: @hpjaj @jholton

Reply:
@zurbergram - Is updating the data in a separate `rake` task an option?  I personally favor updating data in `rake` tasks over migrations, if possible.  That would mitigate against all concerns I believe.  You could run the migration as:

```ruby
def change
  add_column :institutions, :approved, :boolean, null: false, default: false
end
```

And then have a separate `rake` task that updates the records to `approved: true`, that you could scope to `Institution` records where the `created_at` is `<=` to the time of the migration (potentially as a passed value through `args`).

Reply:
That would keep the migration clean.
This is essentially just a SQL UPDATE that is only pertinent to the existing database.  That is, it is inconsequential for anyone running all migrations on a new setup.
@hpjaj how would the rake task be executed?  If you have to ssh to the servers to do so anyway, and since this is just a one-time SQL UPDATE, you could consider it even cleaner to not write the rake task at all and just run `InstitutionModel.update_all(approved: true)` in a Rails console in each environment after deployment.
Thoughts?

Reply:
Agreed!  I like both of those options (rake or live update).

Reply:
@zurbergram, sorry for all the back-and-forth on this.  If you can remove the the code related to the data update from the migration, I can help you with getting the data updates done right after the deployments happen.

Reply:
rake task added then removed once i saw this 😄